tests/gds_kernel_latency.c

```diff
@@ -193,8 +193,22 @@ static int pp_connect_ctx(struct pingpong_context *ctx, int port, int my_psn,
 
 	ctx->ah = ibv_create_ah(ctx->pd, &ah_attr);
 	if (!ctx->ah) {
-		fprintf(stderr, "Failed to create AH\n");
-		return 1;
+		union ibv_gid dgid;
```

Comment:
@ferasd could you please review this change ?
as a context, it is required to run this test in RoCE mode.